assets/panel/debugger.properties

```diff
@@ -293,9 +263,9 @@ editor.addConditionalBreakpoint=Add Conditional Breakpoint
 # input element inside ConditionalPanel component
 editor.conditionalPanel.placeholder=This breakpoint will pause when the expression is true
 
-# LOCALIZATION NOTE(editor.jumpToMappedLocation1): Context menu item
+# LOCALIZATION NOTE(editor.jumpToMappedLocation): Context menu item
 # for navigating to a source mapped location
-editor.jumpToMappedLocation1=Jump to %S location
+editor.jumpToMappedLocation=Jump to %S location
```

Comment:
This should not be reverted, since the previous version of the string used %s (lowercase), we want to reforce translation: https://hg.mozilla.org/mozilla-central/file/default/devtools/client/locales/en-US/debugger.properties#l256